tests/admin.test.ts

```diff
@@ -117,3 +118,83 @@ describe('retroactive attendance submission', () => {
     expect(staffActivityResponse.activity[1].type).toEqual(ActivityType.ATTEND_EVENT_AS_STAFF);
   });
 });
+
+describe('email retrieval', () => {
+  test('gets all the emails of stored users', async () => {
+    const conn = await DatabaseConnection.get();
+    const users = UserFactory.create(5);
+    const emails = users.map((user) => user.email.toLowerCase());
+    const [proxyUser] = UserFactory.with({ accessType: UserAccessType.ADMIN });
+
+    await new PortalState()
+      .createUsers([...users, proxyUser])
+      .write();
+
+    const adminController = ControllerFactory.admin(conn);
+    const response = await adminController.getAllEmails(proxyUser);
+    expect([...emails, proxyUser.email].sort()).toEqual(response.emails.sort());
```

Comment:
nit: you can inline `adminController` since it's only used once:
```ts
const getAllEmailsResponse = await ControllerFactory.admin(conn).getAllEmails(admin);
```

Reply:
Resolved in 5fc6a9a. There are a few instances I inlined as well.